lib/std/os.zig

```diff
@@ -219,7 +219,7 @@ pub var environ: [][*:0]u8 = undefined;
 /// Populated by startup code before main().
 /// Not available on Windows. See `std.process.args`
 /// for obtaining the process arguments.
-pub var argv: [][*:0]u8 = undefined;
+pub var argv: [][*:0]u8 = if (builtin.os.tag != .windows) undefined else @compileError("argv isn't supported on Windows: use std.process.args instead");
```

Comment:
It seems to me that this should be a switch, zig can target other platforms besides windows that don't have argv/aren't posixy. For exampl, WASI.

Reply:
Yep, it should be. I'll get that changed in a moment.